### ISSUE TITLE
Unable to download File Dataset

### ISSUE THREAD

**Issue by michalmar** [Azure/azureml-sdk-for-r#405]
I am unable to download File dataset to Compute Instance FS - I getting "Kernel died" error.

**To Reproduce**
I am getting registered dataset and then trying to download 
ds <- get_dataset_by_name(ws, name='data')
download_from_file_dataset(dataset = ds, overwrite = FALSE)

**Expected behavior**
file would be downloaded to the FS

**Screenshots**
![image](https://user-images.githubusercontent.com/16072588/98728958-76ccc880-239a-11eb-8b36-8e312c8f7711.png)

**Additional context**
azuremlsdk	1.10.0

**Comment by revodavid** [comment]
Installing the latest version of reticulate from github may resolve this issue for you.

**Comment by diondrapeck** [comment]
@michalmar - did updating reticulate solve your issue?

**Comment by michalmar** [comment]
@diondrapeck No, it didn't, I tried with `install.packages("reticulate")` but the error is the same.
